bin/polyfill-service

```diff
@@ -18,6 +18,10 @@ var startService = require('../service');
 var argv = minimist(process.argv.slice(2));
 var port = argv.port || Number(process.env.PORT) || 3000;
 
+process.on('uncaughtException', (err) => {
```

Comment:
Seems sentry actually already logs it

https://app.getsentry.com/nextftcom/polyfill-service-dev/issues/126700489/